transfermethodui/src/androidTest/java/com/hyperwallet/android/ui/transfermethod/SelectTransferMethodTest.java

```diff
@@ -446,4 +482,118 @@ public void testSelectTransferMethod_verifyDefaultsToUSWhenUserProfileDoesNotHav
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText("Wire Transfer")));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesPayPalTransferMethodSelection() {
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("user_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_keys_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_fields_bank_card_response.json")).mock();
+
+        mActivityTestRule.launchActivity(null);
+
+        onView(withId(R.id.select_transfer_method_types_list))
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText("PayPal Account")), click()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText("PayPal Account")));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesLongClickOnTransferMethod() {
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("user_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_keys_response.json")).mock();
+        mMockWebServer.mockResponse().withHttpResponseCode(HTTP_OK).withBody(sResourceManager
+                .getResourceContent("successful_tmc_fields_bank_card_response.json")).mock();
+
+        mActivityTestRule.launchActivity(null);
+
+        onView(withId(R.id.select_transfer_method_types_list))
+                .perform(RecyclerViewActions.actionOnItem(withChild(withText(R.string.bank_account)), longClick()));
+        onView(allOf(instanceOf(TextView.class), withParent(withId(R.id.toolbar)))).check(
+                matches(withText(R.string.title_add_bank_account)));
+
+        verify(mHyperwalletInsight, atLeastOnce()).trackClick(any(Context.class), anyString(), anyString(), anyString(),
+                ArgumentMatchers.<String, String>anyMap());
+    }
+
+    @Test
+    public void testSelectTransferMethod_verifyClickEventCreatedWhenUserMakesDoubleClickOnTransferMethod() {
```

Comment:
You can remove this test as well, as it is not a critical.

Reply:
Removed.